src/components/network-chart/NetworkChartRow.js

```diff
@@ -63,18 +255,25 @@ class NetworkChartRow extends React.PureComponent<NetworkChartRowProps, State> {
       return null;
     }
     const itemClassName =
-      evenOddClassName + ' networkChartRowItem ' + networkPayload.status;
+      evenOddClassName +
+      ' networkChartRowItem ' +
+      this._identifyType(marker.name);
 
     return (
       <section className={itemClassName}>
-        <div className="networkChartRowItemLabel">{marker.name}</div>
+        <div
+          className="networkChartRowItemLabel"
+          onDoubleClick={this._onDoubleClick}
+        >
+          {this._shortenURI(marker.name)}
+        </div>
         <div
           className="networkChartRowItemBar"
           style={markerStyle}
           onMouseEnter={this._hoverIn}
           onMouseLeave={this._hoverOut}
         >
-          &nbsp;
+          <NetworkChartRowBar marker={marker} />
```

Comment:
```suggestion
          <NetworkChartRowBar marker={marker} networkPayload={networkPayload} />
```
This needs the payload passed in as well.